src/translations/es.json

```diff
@@ -469,7 +469,7 @@
   "PaymentMethodsForm.billingDetailsNamePlaceholder": "Escribe el nombre…",
   "PaymentMethodsForm.paymentCardDetails": "Datos de la tarjeta de crédito",
   "PaymentMethodsForm.genericError": "Ha habido algún problema con la información de pagos. Por favor, inténtalo de nuevo.",
-  "PaymentMethodsForm.handleCardPaymentError": "No hemos podido autenticar el método de pago. Por favor checa los detalles de autenticación e intenta de nuevo.",
+  "PaymentMethodsForm.confirmCardPaymentError": "No hemos podido autenticar el método de pago. Por favor checa los detalles de autenticación e intenta de nuevo.",
```

Comment:
Could you check that WTI keeps up with these?
I'm not sure if it fetches these changes automatically or if they need to be changed manually there. I just have a feeling that last time there was strange keys that contained english - and it could be because we have renamed translation keys only on Github.

Reply:
@Gnito It seems that the renamed keys are showing in WTI as new keys, so it doesn't keep up with renaming but there should not be keys containing English either. Should I do the manual updates (as only the key changed and not the content) or should they wait for translators to check them?

Reply:
Since there's just one key, I guess you could copy-paste the content in WTI. (It should end up to translator's proofreading queue)